package.json

```
@@ -36,6 +36,7 @@
     "@changesets/changelog-github": "0.4.8",
     "@changesets/cli": "2.26.2",
     "@types/jest": "29.5.5",
+    "@types/lodash.clonedeep": "^4.5.7",
```

The change I'm suggesting here requires a rerun of `npm i` to update the lockfile.

```suggestion
    "@types/lodash.clonedeep": "4.5.7",
```